images/linux/scripts/installers/python.sh

```diff
@@ -40,4 +40,7 @@ if isUbuntu18 || isUbuntu20 ; then
     fi
 fi
 
+# Adding this dir to PATH will make installed pip commands are immediately available.
+prependEtcEnvironmentVariable "$HOME/.local/bin"
```

Comment:
This won't work. we have different users on ADO and GA images.

Reply:
This code will add the following line `/home/packer/.local/bin` to `/etc/environments`.
After deployment, we will have the following users: `/home/vsts` (ADO) and `/home/runner` (GA) so this line will do nothing.

Please reevaluate if we really need to add this folder to PATH. If it is necessary, we can:
1) move pip cache under non-user directory
2) use post-build scripts

Reply:
$HOME/.local/bin added to skel for the user. Thank you for the meaningful reply.